src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,15 @@
-export const TodoList = () => {};
+import React from 'react';
+import { TodoInfo } from '../TodoInfo';
+import { Todo } from '../../types';
+interface Props {
+  todos: Todo[];
+}
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  return (
+    <section className="TodoList">
+      {todos.map((todo, index) => (
+        <TodoInfo todo={todo} key={`${todo.id}_${index}`} />
```

Comment:
```suggestion
        <TodoInfo todo={todo} key={todo.id} />
```

Id should be unique, so you don't need to do this